proto/iotics/api/input.proto

```diff
@@ -72,6 +79,82 @@ message DeleteInputResponse {
 }
 // ---------------------------------------
 
+// CreateInputRequest describes a created input.
+message CreateInputRequest {
+  // CreateInputRequest payload.
+  message Payload {
+    // Input identifier string representation (simple string) (unique within the scope of a twin identifier's input set)
+    string id = 1;
+  }
+
+  // Arguments describes the mandatory arguments to identify the twin owning this input
+  message Arguments {
+    // Identifier of the twin owning this input
+    TwinID twinId = 1;
+  }
+
+  // CreateInputResponse headers
+  Headers headers = 1;
+  // CreateInputRequest arguments
+  Arguments args = 2;
+  // CreateInputResponse payload
+  Payload payload = 3;
+}
+// ---------------------------------------
+
+// CreateInputResponse describes a created input.
+message CreateInputResponse {
+  // CreateInputResponse payload.
+  message Payload {
+    // Created input
+    InputID inputId = 1;
+  }
+
+  // CreateInputResponse headers
+  Headers headers = 1;
+  // CreateInputResponse payload
+  Payload payload = 2;
+}
+// ---------------------------------------
+
+// UpdateInputRequest is used to update an input.
+message UpdateInputRequest {
+  // UdpateInputRequest payload used to update one or more field
+  message Payload {
+    // Values describe individual data items to add/remove
+    repeated Values values = 1;
+    // Properties describe individual properties to add/remove. Note only cusom properties can be modified here.
+    repeated PropertyUpdate properties = 2;
```

Comment:
The PropertyUpdate [already](https://github.com/Iotic-Labs/api/blob/817c200ede2946f8f03b95a195c681faf52cffa7/proto/iotics/api/common.proto#L217-L232) supports multiple actions - the repetition is within it.
But.. I notice that its doc string says:
> describes the update of **a twin** properties

.. but it can be about a Twin's, Feed's or Input's properties now - so maybe at the same time you could update it to reflect that.